dashboard/test/ui/features/star_labs/signin_callout.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Viewing and dismissing the login callout
+
+Scenario: Should see callout on 20-hour farmer lesson
+  Given I am on "http://studio.code.org/s/20-hour/stage/9/puzzle/1?noautoplay=true"
+  And I rotate to landscape
+  And I wait for the page to fully load
+  And element ".modal-backdrop" is visible
```

Comment:
Is there a more "direct" element that we can check to see if the callout is present?  (".modal-backdrop" seems a bit tangential to the callout and also might be confused by other modal backdrops?)

Reply:
Yes... this is a good point. I think I initially chose it because it was something that would catch a click anywhere (and not just on the actual callout). But I could figure out how to rename a class somewhere in here!

Reply:
Done- I renamed this class to "login-callout" so it (hopefully) won't conflict with anything else!